exercises/practice/acronym/acronym_test.rb

```diff
@@ -4,36 +4,46 @@
 class AcronymTest < Minitest::Test
   def test_basic
     # skip
-    assert_equal 'PNG', Acronym.abbreviate('Portable Network Graphics')
+    assert_equal "PNG", Acronym.abbreviate("Portable Network Graphics")
```

Comment:
I prefer single quotes... but... it should be consistent throughout the repository, either way.